### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by wanjunlei**
Signed-off-by: wanjunlei <wanjunlei@kubesphere.io>

### CODE REVIEW

File: docs/crds/router.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+- `alertSelector` - A label selector used to match alert. The matched alert will send to the `receivers`.
+- `receivers.name` - The name of receivers which notifications will send to.
+- `receivers.regexName` - A regular expression to match the receiver name.
+- `receivers.selector` - A label selector used to select receivers.
+- `type` - The type of receiver, known values are dingtalk, email, feishu, pushover, sms, slack, webhook, WeChat.
+
+## Examples
+
+A router that route all notifications to the all receivers of tenant `user1`.
```
Comment (Bettygogo2021) [inline docs/crds/router.md:37]: route>>>routes

File: docs/crds/router.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+metadata:
+  name: router1
+spec:
+  receivers:
+    selector:
+      matchLabels:
+        tenant: user1
+```
+
+A router that route cluster-level notifications to the email receivers `user1`.
```
Comment (Bettygogo2021) [inline docs/crds/router.md:51]: route>>>routes

File: docs/crds/silence.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+spec:
+  matcher:
+    matchExpressions:
+      - key: namespace
+        operator: In
+        values:
+        - test
+```
+
+A silence that mute all notifications in namespace `test` and activate at `2022-02-29T00:00:00Z` for 24 hours.
```
Comment (Bettygogo2021) [inline docs/crds/silence.md:42]: 1. mute>>> mutes
2. activate >>> is activated

File: docs/template.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
+# Template
+
+## Overview
+
+The notifications sent to receivers constructed via templates. The Notification Manager comes with default templates, but they can also customize.
```
Comment (Bettygogo2021) [inline docs/template.md:5]: sent >>> are sent
but they can also customize>>> which also support customization

File: docs/template.md [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+| webhook  | &nbsp;    | webhook.default.message | &nbsp;                   |
+
+> You can find these default templates at [here](../config/samples/template.yaml).
+
+> `webhook.default.message` is a special template, it will not generate notifications, 
+> notification manager will serialize the data to json and send it to webhook.
+
+## Customize template
+
+The notification templates based on the [Go templating](https://pkg.go.dev/text/template) system.
```
Comment (Bettygogo2021) [inline docs/template.md:139]: based on >>> is based on

File: docs/template.md [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+| Name              | Type            | Notes                                           | 
+|-------------------|-----------------|-------------------------------------------------|
+| Alerts            | [Alert](#Alert) | List of alert.                                  |
+| GroupLabels       | [KV](#KV)       | Labels used to group alerts in `Alerts`.        |
+| CommonLabels      | [KV](#KV)       | The labels common to all of the alerts.         |
+| CommonAnnotations | [KV](#KV)       | The annotations common to all of the alerts. |
+
+The `Data` type exposes a function for getting status.
+
+- `Status` returns the status of `Alerts`. if there has a firing alert in `Alerts`, it returns `firing`, else return `resolved`.
```
Comment (Bettygogo2021) [inline docs/template.md:197]: else return >>> else it returns

File: docs/template.md [hunk 7]
```diff
@@ -0,0 +1,309 @@
...
+| toUpper      | string                      | string  | Converts all characters to upper case.                                   |
+| toLower      | string                      | string  | Converts all characters to lower case.                                   |
+| join         | sep string, s []string      | string  | Similar to `strings.Join`.                                               |
+| match        | pattern, string             | bool    | Match a string using Regexp.                                             |
+| reReplaceAll | pattern, replacement, text  | string  | Regexp substitution.                                                     |
+| translate    | string                      | string  | Translate the string, see [Multilingual support](#Multilingual-support). |
+
+## Multilingual support
+
+Notification manager supports custom the notification language.
```
Comment (Bettygogo2021) [inline docs/template.md:273]: custom the notification language>>>language customization

File: docs/template.md [hunk 8]
```diff
@@ -0,0 +1,309 @@
...
+metadata:
+  name: notification-manager
+spec:
+  template:
+    language: zh-cn
+    languagePack:
+      - name: zh-cn
+        namespace: kubesphere-monitoring-system
+```
+The `languagePack` is a list of `ConfigmapKeySelector`. The configmap likes this.
```
Comment (Bettygogo2021) [inline docs/template.md:287]: likes this: is like this.

**Review by Bettygogo2021 — COMMENTED** [review]
Hi, some issues are repetitive, so you can search all of them and batch modify them.
For any doubts, feel free to contact me.

File: docs/crds/notification-manager.md [hunk 9]
```diff
@@ -0,0 +1,274 @@
+# NotificationManager CRD
```
[the remark this answers is not in the document]
Comment (wanjunlei) [reply docs/crds/notification-manager.md:1]: The `NotificationManager` is the name of CRD.

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

**Review by benjaminhuo — APPROVED** [review]

**Comment by benjaminhuo** [comment]
This is a huge improvement!

**Pull request merged**